src/_includes/common/configuration-intro.md

```diff
@@ -0,0 +1,31 @@
+{% comment %}
+Guideline: This page is common to all SDKs; it is stored in the common folder, nested under _includes/common. To use, 
+
+1. Add a folder with the name of the platform you are documenting to the _documentation/sdks structure (for example, _documentation/sdks/javascript) 
+2. Create a new config folder and new intro.md file in _documentation/sdks/<platform-name> 
+3. Create the defined `include` statements and add them to the intro.md file
+
+If you have questions, please ask Fiona or Daniel. 
+
+**The objective for this page is that a developer can view easily the configuration options for the SDK; each page _must_ have a description that includes a summary of what the page provides to the developer. Simply linking the page is insufficient.**
+{% endcomment %}
+
+Sentry's {{ include.sdk_name }} SDK includes many configuration options that are automatically set. You can configure your SDK using the options outlined in these pages:
+
+- **[Basic Options](/sdks/{{ include.root_link }}/config/basics)** 
+
+    Learn more about how to configure the {{ include.sdk_name }} SDK. These options are set when the SDK is first initialized, passed to the `init()` as an object. In addition, this page discusses the Hooks `beforeSend` and `beforeBreadcrumb` in addition to transport options.
```

Comment:
The second sentence here sounds odd to me. "In addition, ... in addition to..."

Reply:
I'm meaning it as "This page also discuss Hooks" - I'll clarify.